src/Misc/layoutroot/run.sh

```diff
@@ -53,6 +53,34 @@ runWithManualTrap() {
     done
 }
 
+# update ca certificates in case they are injected with a volume mount
+function updateCerts() {
+    local sudo_prefix=""
+    local user_id=`id -u`
+
+    if ! [ $user_id -eq 0 ]; then
+        if [[ ! -x "$(command -v sudo)" ]]; then
+            echo "Warning: failed to update certificate store: sudo is required but not found"
+            return 1
```

Comment:
`return 1` or just `return`?

Reply:
I felt like the function should report an error with an exit code even if we ignore it
It should signal the failure anyway, and it is up to the caller to ignore it or use it to terminate the execution